### ISSUE TITLE
Log more information about an object as extra.

### ISSUE THREAD

**Issue by jensens** [collective/collective.fingerpointing#69]
Why? -> current ``repr`` is poor and makes search difficult.

Better extra: ``type=Image path=/Plone/foo/bar uid=01f4a867a5e3487e81603c0fdb222018``

Implementation should be pluggable so one can add own ``key=value`` pairs.

**Comment by hvelarde** [comment]
sound good; this is related with #10 but I have no strong opinion here; I would like to keep the implementation simple, but I'm open to new ideas.

**Comment by jensens** [comment]
Simple is good. I.e  I would not want to lookup adapters here, also it must be fast. Its just a rough idea.

**Comment by runyaga** [comment]
I agree with @jensens.  We might be using this package soon.  We would need key=value pairs because we need to pass in a note="contextual humane note about action".
